### PR TITLE
fix/AUT-2015/Hotspot show set score in spots

### PR DESCRIPTION
**Pull request by bziondik**
Related to: https://oat-sa.atlassian.net/browse/AUT-2015

**How to test:**

- Go to Item Authoring
- Add Hotspot
- Add spots
- Map response selected in response
- Adding scores for spots
- Switch to Question state
- Switch to Response state
- Spots should contain right score

image

### CODE REVIEW

File: views/js/qtiCreator/widgets/interactions/hotspotInteraction/states/Map.js [hunk 1]
```diff
@@ -61,93 +76,98 @@ define([
...
-        _.forEach(interaction.getChoices(), function(choice){    
-            var shape = interaction.paper.getById(choice.serial);
-            var $popup = grahicScorePopup(interaction.paper, shape, $imageBox, isResponsive);
-            var score = mapEntries[choice.id()] || response.mappingAttributes.defaultValue || '0'; 
+        _.forEach(interaction.getChoices(), function (choice) {
+            const shape = interaction.paper.getById(choice.serial);
+            const $popup = grahicScorePopup(interaction.paper, shape, $imageBox, isResponsive);
+            let scoreDefault = false;
+            if (mapEntries[choice.id()]) {
+                scoreDefault = false;
```
Comment (marpesia) [inline views/js/qtiCreator/widgets/interactions/hotspotInteraction/states/Map.js:130]: I cannot see where this variable could be `true`, and it is always `false`.
Does it make sense to this conditional?

Comment (bziondik) [reply views/js/qtiCreator/widgets/interactions/hotspotInteraction/states/Map.js:130]: Fixed @marpesia

**Review by marpesia — APPROVED** [review]
Works as expected. Good job!

**Review Checklist**
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [ ] Acceptance criteria are respected
- [ ] Pull request title and description are meaningful
- [ ] Pull request's target is not `master`
- [ ] Commits are following [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/)
- [ ] Commits messages are meaningful
- [ ] Commits are atomic
- [ ] Changelog is updated according to changes (if applicable)
- [ ] Documentation is updated according to changes (if applicable)

**Review by meefox — APPROVED** [review]
well done 
Screenshot 2022-03-25 at 10 46 18

**Pull request merged**